src/Toolkit.Forms/NetStandardStubs.cs

```diff
@@ -105,7 +105,7 @@ internal class TimeSlider
         public Task InitializeTimePropertiesAsync(ITimeAware timeAwareLayer) => throw new NotImplementedException();
         public bool StepForward(int timeSteps = 1) => throw new NotImplementedException();
         public bool StepBack(int timeSteps = 1) => throw new NotImplementedException();
-        public event EventHandler<TimeExtentChangedEventArgs> CurrentExtentChanged;
+        public event EventHandler<TimeExtentChangedEventArgs>? CurrentExtentChanged;
     }
 }
```

Comment:
Was **Forms.SymbolDisplay** class missed? Looks like its Symbol property should be nullable (in the stub and in the implementation too).